### ISSUE TITLE
Upgrading to 0.9.0 is throwing an error

### ISSUE THREAD

**Issue by jasikpark** [delucis/astro-embed#171]
```
Cannot read properties of undefined (reading 'object')
  Stack trace:
    at file:///Users/calebjasik/Git/jasik-2.0/.netlify/build/chunks/post_C9dswsZq.mjs:13678:7
    at ModuleJob.run (node:internal/modules/esm/module_job:268:25)
    at async render (file:///Users/calebjasik/Git/jasik-2.0/.netlify/build/chunks/site-config-vars_BmO8zSp5.mjs:611:28)
    at async callComponentAsTemplateResultOrResponse (file:///Users/calebjasik/Git/jasik-2.0/node_modules/.pnpm/astro@4.16.10_@types+node@22.9.0_lightningcss@1.28.1_rollup@4.21.2_typescript@5.6.3/node_modules/astro/dist/runtime/server/render/astro/render.js:90:25)
    at async renderPage (file:///Users/calebjasik/Git/jasik-2.0/node_modules/.pnpm/astro@4.16.10_@types+node@22.9.0_lightningcss@1.28.1_rollup@4.21.2_typescript@5.6.3/node_modules/astro/dist/runtime/server/render/page.js:31:24)
 ELIFECYCLE  Command failed with exit code 1.
```

Upgrading to 0.9.0 for bluesky post support is throwing an error on build. Strangely not when in dev mode though. At first I thought it was b/c I hadn't enabled content layers, but that didn't seem to fix anything.

**Comment by delucis** [comment]
Hmm. Any chance you can share a reproduction?

**Comment by jasikpark** [comment]
Not too minimal, but here you go: https://github.com/jasikpark/astro-embed-repro-171

**Comment by delucis** [comment]
Ok, so this is weird, but are you using `zod` directly anywhere in the real code base? The error above (`reading 'object'`) looks to be an issue in JS using `z.object({})` where Zod is bundled incorrectly leading to `z` being undefined. Not sure exactly _why_, but uninstalling the direct Zod dependency (which is not used in your repro) made builds work again.

If you _are_ using Zod yourself directly, you could try replacing those imports to use Astro’s re-export from the `astro/zod` module.

**Comment by jasikpark** [comment]
I was using it in Layout.astro, but replacing the imports like you said works. This is very strange, but honestly I'm not motivated to discover why things are broken. Thanks for helping me figure it out!

**Comment by JeroenReumkens** [comment]
I just ran into the same error after days of trying to find where this error was coming from. Eventually nailed it down to this package too. 

I fixed it by changing my import to `@astro-community/astro-embed-twitter`. 

Not sure if this is helpful at all to you, but I thought it would at least be good to know that there's multiple occurrences of this issue, and that it's very hard to track down 😅
